### PR TITLE
Add ManaTEE project

### PR DESCRIPTION
**Pull request by riaankleinhans**

### CODE REVIEW

File: Projects/README.md [hunk 1]
```diff
@@ -158,6 +159,24 @@ Keystone is an open-source project for building trusted execution environments (
...
+
+ManaTEE is an open-source project for easily building and deploying data collaboration framework to the cloud using trusted execution environments (TEEs). It allows users to easily collaborate on private datasets without leaking privacy of individual data.
+
+**Status:** Incubation
+
+### Links
+
+* [Project detail]()
+* [GitHub](https://github.com/manatee-project/manatee/)
+* [Proposal](./Occlum/occlum-accepted_proposal.pdf)
```
Comment (mssun) [inline Projects/README.md:174]: Incorrect link to the proposal.

File: Projects/README.md [hunk 2]
```diff
@@ -158,6 +159,24 @@ Keystone is an open-source project for building trusted execution environments (
...
+    * Accepted by TAC: 2024-07-25
+    * Accepted by Governing Board: 2024-08-10
+* [Charter](./ManaTee/ManaTEE-Technical-Charter-Final-9-26-2024.pdf)
+* [Artwork](https://github.com/confidential-computing/artwork#occlum)
```
Comment (mssun) [inline Projects/README.md:178]: Incorrect link.

File: Projects/README.md (hunk 1 above)
Comment (riaankleinhans) [reply Projects/README.md]: thanks, updated

File: Projects/README.md (hunk 2 above)
Comment (riaankleinhans) [reply Projects/README.md]: thanks, updated

**Pull request merged**